src/Widgets/HeaderBar.vala

```diff
@@ -68,8 +76,6 @@ namespace Maya.View {
             pack_end (menu_button);
             pack_end (contractor);
 
-            button_add.clicked.connect (() => add_calendar_clicked ());
-            button_today.clicked.connect (() => on_menu_today_toggled ());
```

Comment:
Yeah since the buttons have an action name associated, we don’t need to manually handle the signals anymore. Having an action automatically sets sensitivity and handles the activation signal